libs/lib_parsing/Origin.ml

```diff
@@ -18,15 +18,15 @@
 type t =
   | File of Fpath.t
   | GitBlob of {
-      sha : Git_wrapper.sha;
-      paths : (Git_wrapper.sha * Fpath.t) list;
+      blob : Git_wrapper.hash;
```

Comment:
why the rename from sha to blob? It was clearer before that the field was containing a ref/hash to the blob,
not the actual blob.

Reply:
I switched it to a blob type before, then back to the hash and forgot to fix this, tks